pkg/flag/options.go

```diff
@@ -83,6 +85,20 @@ type Options struct {
 	DisabledAnalyzers []analyzer.Type
 }
 
+// Align takes consistency of options
+func (o *Options) Align() {
+	if o.Format == report.FormatSPDX || o.Format == report.FormatSPDXJSON {
+		log.Logger.Debug(`"--format spdx" and "--format spdx-json" disable security checks`)
+		o.SecurityChecks = nil
+	}
+
+	// Vulnerability scanning is disabled by default for CycloneDX.
+	if !viper.IsSet(SecurityChecksFlag.ConfigName) && o.Format == report.FormatCycloneDX {
```

Comment:
it doesn't matter, just a note:
i'd like to change these statements:
```suggestion
	if o.Format == report.FormatCycloneDX && !viper.IsSet(SecurityChecksFlag.ConfigName) {
```

Reply:
Fixed
https://github.com/aquasecurity/trivy/pull/2541/commits/7a885bc4dcc6ea49a1929ef7fbc17519140bf8be